members_and_sponsors.md

```diff
@@ -8,6 +8,7 @@ Sponsors:
 - The D. E. Shaw group
 - Quansight
 - LG Electronics
+- Bodo
```

Comment:
@ehsantn just to make sure: do you prefer "Bodo" or maybe "Bodo, Inc" or "Bodo AI"? Further down we list the software project, here it's the company.